src/main/scala/com/twitter/diffy/DiffyServiceModule.scala

```diff
@@ -70,6 +70,9 @@ object DiffyServiceModule extends TwitterModule {
   val skipEmailsWhenNoErrors =
     flag[Boolean]("skipEmailsWhenNoErrors", false, "Do not send emails if there are no critical errors")
 
+  var httpsPort =
+    flag[String]("httpsPort", "443", "Port to be used when using HTTPS as a protocol")
```

Comment:
Rather than add a new flag, let's use the existing "proxy.port" flag. The "protocol" flag can be set to "https" and should be sufficient to create a SimpleHttpsDifferenceProxy and bind it to the "proxy.port".